backend/src/main/java/com/google/sps/data/Participant.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+/** A user who want to be matched. */
+public final class Participant {
+
+  private final long id;
+  private final String ldap; // Google username
+  private final long timeAvailableUntil; // Time user is available until
```

Comment:
The timestamp is when the participant is "created" in AddParticipantServlet. So that's the beginning of their availability range. If you're talking about the beginning of the meeting time, that will be calculated when we create the calendar event based on the duration and the time that the match is found.

Reply:
I think it would be good to distinguish between when the entry was created and the beginning of the participant's availability range. In the future if you want to allow users to say "I'm not free now, but I would like to set something up two hours from now" then supporting a distinct start of the range now would make that easier.

Reply:
I wrote this down as a todo after these PRs get merged. Because my PRs are all interdependent, it would be too complicated to refactor everything in this PR, but I will definitely do this in the next one.